image_provisioner/inventory/hosts

```diff
@@ -0,0 +1,55 @@
+# vim:ft=ansible:
+localhost ansible_connection=local ansible_python_interpreter=/usr/bin/python
+
+[all:vars]
+# AWS vars
+aws_access_key=
+aws_secret_key=
+region=us-west-2
+key_name=
+group_id=
+instance_type=m4.xlarge
+vpc_subnet_id=
+base_image=
+# repo install
+internal_image=false
+rhel_int_repo_hostname=
+# collectd install
+config_type=ose
+collectd_interval=10
+collectd_status=started
+graphite_host=
+graphite_prefix=ose
+# clone repos
+github_token=
+# pbench config
+image_tag=2016-09-09
```

Comment:
I tried to figure out a way to get this to be auto-calculated at runtime and the feedback I got was:

You could assemble the timestamp using facts or using a lookup, same goes for expanding the home directory.

http://docs.ansible.com/ansible/playbooks_variables.html#information-discovered-from-systems-facts
http://docs.ansible.com/ansible/playbooks_lookups.html